templates/download/qualcomm-iot/tabs/development-board-tab.html

```diff
@@ -34,6 +34,16 @@ <h3 class="p-heading--5">Ubuntu 24.04 LTS</h3>
       </p>
       <div class="p-section--shallow">
         <div class="row">
+          <div class="col-6 col-medium-4 col-small-4">
+            <hr class="p-rule--muted" />
+          </div>
+          <div class="col-3 col-medium-2 col-small-2">
+            <p class="p-heading--5">Download Ubuntu Desktop 24.04</p>
```

Comment:
```suggestion
            <p class="p-heading--5">Ubuntu Desktop 24.04</p>
```
Please remove 'Download' from the text preceding all 3 download buttons to match the other tabs